spring-cloud-kubernetes-fabric8-discovery/src/main/java/org/springframework/cloud/kubernetes/fabric8/discovery/KubernetesCatalogWatch.java

```diff
@@ -66,7 +67,7 @@ public void setApplicationEventPublisher(ApplicationEventPublisher publisher) {
 		this.publisher = publisher;
 	}
 
-	@Scheduled(fixedDelayString = "${spring.cloud.kubernetes.discovery.catalogServicesWatchDelay:30000}")
```

Comment:
I extracted this String `spring.cloud.kubernetes.discovery.catalogServicesWatchDelay` (and some related to it) to commons, so that all 3 clients use the same thing. We have been doing that across multiple other PRs in various places, needed of course for complete alignment in all 3 clients